### PR TITLE
[ON HOLD] Adding log rotate

### LINKED ISSUE #510: Implement `logrotate` to every dappnode logs files
Implement `logrotate` to every dappnode log file:
- `/usr/src/dappnode/logs*`
- `/usr/src/dappnode/DNCORE/userAction.logs`

### PR DESCRIPTION
**Pull request by 3alpha**
Resolves #510 

This PR adds log rotation to DAppNode logs and prevents them to grow oversized.

### CODE REVIEW

**Review by dsimog01 — COMMENTED** [review]
LGTM, but I do not have much knowledge about log rotation config.

File: scripts/dappnode_install.sh [hunk 1]
```diff
@@ -289,6 +289,21 @@ installExtraDpkg() {
     fi
 }
 
+add_log_rotate_conf() {
```
Comment (pablomendezroyo) [inline scripts/dappnode_install.sh:292]: the log files `.log` are okey to be compressed. But what about the `userActionLog.json` file, this file is consumed by the dappmanager. Can it be consumed even compressed?

**Review by pablomendezroyo — CHANGES_REQUESTED** [review]

File: scripts/dappnode_install.sh (hunk 1 above)
Comment (3alpha) [reply scripts/dappnode_install.sh:292]: I would guess no. But idea behind this is that it compresses old files and lets dappmanager consume only latest (in this case less that week old) logs. But we should test thoroughly this to see if it really works.

Comment (pablomendezroyo) [reply scripts/dappnode_install.sh:292]: Lets try to have it ready by next devs meeting

Comment (pablomendezroyo) [reply scripts/dappnode_install.sh:292]: status?

Comment (3alpha) [reply scripts/dappnode_install.sh:292]: On my machine it behaved fine, @dsimog01 was testing it as well. If it worked for him, we should merge this.

**Comment by dsimog01** [comment]
@3alpha can this be merged?

**Comment by pablomendezroyo** [comment]
@3alpha ping

**Comment by 3alpha** [comment]
We should close this one because it doesn't work on JSON logs which are the main issue.

**Pull request closed without merging**